pkg/detectors/bombbomb/bombbomb.go

```diff
@@ -20,7 +21,7 @@ var (
 	client = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"bombbomb"}) + `\b([a-zA-Z0-9-._]{704})\b`)
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"bombbomb"}) + `\b(eyJ[A-Za-z0-9_-]{10,}\.[A-Za-z0-9_-]{10,}\.[A-Za-z0-9_-]{10,})\b`)
```

Comment:
The generated secrets are JWTs, and I recommend using a standard JWT regex pattern: `\b[A-Za-z0-9-_]+\.[A-Za-z0-9-_]+\.[A-Za-z0-9-_]+\b`. Additionally, I suggest adding this pattern to the common patterns [here](https://github.com/trufflesecurity/trufflehog/blob/86d2c6ded3069d9527b83d7068ac756a91514866/pkg/common/patterns.go) so that it can be utilized by other detectors that look for JWT secrets.

@zricethezav @mcastorina @abmussani thoughts on adding JWT in common patterns? ^